strawberryfields/utils/post_processing.py

```diff
@@ -0,0 +1,341 @@
+# Copyright 2019-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module provides functions for post-processing samples.
+"""
+import numpy as np
+
+
+def number_expectation_pnr(photon_number_samples, modes=None):
+    """The expectation value of the number operator from PNR samples.
+
+    **Example:**
+
+    .. code-block:: python
+
+        samples = np.array([[2, 0],
+                            [2, 2],
+                            [2, 0],
+                            [0, 0]])
+
+    Getting the expectation value for these PNR samples:
+
+    >>> number_expectation_pnr(samples)
+    1.0
+
+    Args:
+        photon_number_samples (ndarray): the photon number samples with a shape
+            of (shots, modes)
+        modes (Sequence): a flat sequence containing indices of modes to get
+            the expectation value for
+
+    Returns:
+        float: the expectation value from the samples
+    """
+    return _samples_expectation(photon_number_samples, modes)
+
+
+def number_variance_pnr(photon_number_samples, modes=None):
+    """The variance of the number operator from PNR samples.
+
+    **Example:**
+
+    .. code-block:: python
+
+        samples = np.array([[2, 0],
+                            [2, 2],
+                            [2, 0],
+                            [0, 0]])
+
+    Getting the variance for these PNR samples:
+
+    >>> number_expectation_pnr(samples)
+    3.0
+
+    Args:
+        photon_number_samples (ndarray): the photon number samples with a shape
+            of (shots, modes)
+        modes (Sequence): a flat sequence containing indices of modes to get
+            the variance for
+
+    Returns:
+        float: the variance from the samples
+    """
+    return _samples_variance(photon_number_samples, modes)
+
+
+def quadrature_expectation_homodyne(homodyne_samples, modes=None):
```

Comment:
As far as I can tell, there is no difference between this function and `number_expectation_pnr` except the docstring?